lib/active_merchant/billing/gateways/mercado_pago.rb

```diff
@@ -196,6 +198,46 @@ def add_payment(post, options)
         post[:payment_method_id] = options[:payment_method_id] if options[:payment_method_id]
       end
 
+      def add_net_amount(post, options)
+        post[:net_amount] = options[:net_amount].to_f if options[:net_amount]
```

Comment:
Since we're adding this field to a JSON object, does the `.to_f` here make a difference?

Reply:
I added this for the case where a user submits the field as a string containing a floating point, i.e. `"net_amount": "5.60"` would be converted to `"net_amount": 5.6` which is what Mercado Pago expects from testing.